index.php

```diff
@@ -173,13 +173,20 @@ function createCountdown() {
 			if (bg != '') { newUrl += '&bg='+encodeURIComponent(bg); }
 			window.location.href = newUrl.replace(/ +/g,'+');
 		}
+		var tzValue = '';
```

Comment:
If the user types invalid input immediately after the page loads (ex. Changes "+3" to "+3abc", their input will be entirely erased instead of going back to "+3"

On line 228, when the user's timezone is acquired, I think something like this would solve that:

    var tzText = (tz >= 0) ? "+" + tz : tz;
    document.querySelector("#utcInput").value = tzText;
    tzValue = tzText;

Reply:
I missed it!